### PR TITLE
Jb/dynamic events

### PR DESCRIPTION
**Pull request by jbouzi12**
This PR is for the new Events Page. Includes a few additions/updates:

- New Events Page
- Updated Folder Structure to account for events route format = `/community/events'
- New `EventsList` component for displaying a lazy-load grid of `Events`
- New `FeatureBlock` component (@melissarudofsky I found another case of the feature block we were both working on on the events page, so I lifted much of your work to create a `FeatureBlock` component)
- Date utility functions copied from `mitx`
- hook to pull events data (not working yet due to CORS issues)

### Next Steps
- get events api working = waiting for Deepak to get back to us to we're allowed to hit the api
- get filtering working = current Filter component hard wired for the `Learn` page. So need a version for Events

### CODE REVIEW

File: src/components/dev-hub/event-list.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        };
+        getData();
+    }, [url]);
+
+    return [events, error];
+};
+
+export const EventsListPreview = () => {
+    let previews = sampleEvents;
+    const [events, error] = useEventData(EVENTS_API);
```
[the remark this answers is not in the document]
Comment (jbouzi12) [reply src/components/dev-hub/event-list.js]: Yes! I'll update this here with a comment.

File: src/components/dev-hub/event-list.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    }
+    @media ${screenSize.largeAndUp} {
+        grid-template-columns: repeat(3, 1fr);
+    }
+`;
+const CenterBlock = styled('div')`
+    text-align: center;
+`;
+
+export const useEventData = url => {
```
[the remark this answers is not in the document]
Comment (jbouzi12) [reply src/components/dev-hub/event-list.js]: Nope I can place this in its own hooks file 👍

File: src/pages/community/events.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            </HeroBanner>
+            <section>
+                <EventsFilter>
+                    {/* TODO: Update filter bar below to filter by other content */}
+                    <FilterBar heading="All Events" />
+                </EventsFilter>
+                {/* TODO: remove below when events_api is working */}
+                <EventsList items={[...sampleEvents, ...sampleEvents]} />
+            </section>
+            <FeatureBlock
```
Comment (greggb) [inline src/pages/community/events.js]: we can kill this

File: src/components/dev-hub/events.js [hunk 4]
```diff
@@ -146,13 +204,25 @@ const DateIcon = ({ date }) => {
...
+        'node_type_attributes'
+    );
+
+    const urlProp = {
+        // internal mdb event urls only contain relative path,
+        // so these must be updated manually
+        href: urlType === 'alias' ? `${MONGODB_WEBSITE}/${url}` : url,
+    };
     return (
         <StyledEvent
```
Comment (greggb) [inline src/components/dev-hub/events.js]: we should add an `_blank` target with `noopener noreferrer` for these

**Pull request merged**